flag.go

```diff
@@ -132,6 +132,14 @@ type VisibleFlag interface {
 	IsVisible() bool
 }
 
+// CategorizableFlag is an interface that allows us to potentially
+// use a flag in a categorized representation.
+type CategorizableFlag interface {
+	VisibleFlag
+
+	GetCategory() string
```

Comment:
Rename to just Category(). golang doesnt recommend using GetXXX

Reply:
What about the conflict with the field name `Category`? 🤔 I agree that `GetXXX` is generally bad style, but this is a language-level wart we're working around.